lib/data-fetcher.js

```diff
@@ -33,7 +43,7 @@ const USER_AGENT = ['Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) Appl
 function fetchMetadataDescription(url) {
   return new Promise((resolve, reject) => {
```

Comment:
From a previous PR, but I think the `new Promise()` can be eliminated here. (It could written `return fetchWithUA(url)…`, with `return $('meta[name="description"]').attr('content');` in the final `.then()`, rather than resolve.)